app/src/main/res/layout/screen_manual_patient_entry.xml

```diff
@@ -55,7 +55,8 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_marginTop="@dimen/spacing_16"
-        android:paddingStart="@dimen/spacing_4">
```

Comment:
I see that the separator was already there. What exactly was the reason for the separator to not be visible?

Reply:
It's because of the margin top and gravity of the container.